include/ck/utility/get_id.hpp

```diff
@@ -15,6 +15,10 @@ __device__ index_t get_warp_local_1d_id() { return threadIdx.x / get_warp_size()
 
 __device__ index_t get_block_1d_id() { return blockIdx.x; }
 
+__device__ index_t get_block_id_z() { return blockIdx.z; } // used in batched gemm
```

Comment:
@asroy What is your opinion for introducing blockIdx.z?

Reply:
I think introducing will make "block id" n-D concept complicated.

If the purpose is to avoid integer division, you can use "merge" transform which will use magic division underneath

Reply:
Removed in latest commit